benchmarks/hmm_semisup/turing.jl

```diff
@@ -44,6 +46,139 @@ n_steps = 4
 test_zygote = false
 test_tracker = false
 
-include("../infer_turing.jl")
```

Comment:
Why do we have a seperate chunk of codes here instead of using `infer_turing.jl`

Reply:
This was an attempt to get the script running as the benchmarking errors due to an initialisation bug.